ingestion/src/metadata/ingestion/ometa/mixins/lineage_mixin.py

```diff
@@ -153,6 +155,8 @@ def _create_lineage_by_table_name(
         This method is to create a lineage between two tables
         """
         try:
+            from_table = str(from_table).replace("<default>", "")
```

Comment:
Why are we using the `<default>` here? From where does it come from?

Reply:
<default> comes from lineage runner itself.
But it should be `<default>.` that needs to be replaced.
<img width="903" alt="image" src="https://user-images.githubusercontent.com/40225091/163995832-9ad8ebd0-a6dd-4713-84b4-6c401e0d58d0.png">

Reply:
@ulixius9 can you check if there is a need to replace the dot as well

Reply:
No replacing dot is not required, in fact if I replace the dot then the [split](https://github.com/open-metadata/OpenMetadata/blob/540f5ca3b50022717894ca23d1b9c04b26e98aee/ingestion/src/metadata/ingestion/ometa/mixins/lineage_mixin.py#L145) logic fails

Reply:
then let's add a constant for this, something like `LINEAGE_RUNNER_DEFAULT` to help understand what we are doing 🙏